sv_SE/LC_MESSAGES/playlist.po

```diff
@@ -4,136 +4,82 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 
 msgid "ADDED_TO_NEW_PLAYLIST"
-msgstr "Added to new Playlist"
+msgstr "Tillagd i ny spellista"
 
 msgid "ADDED_TO_PLAYLIST"
-msgstr "Added to Playlist"
-
-msgid "BACK_TO_CREATORS_ROOM"
-msgstr "Back to Creator's Room"
+msgstr "Tillagd i spellista"
 
 msgid "BACK_TO_FLIPNOTE"
-msgstr "Back to Flipnote"
+msgstr "Tillbaka till Flipnote"
 
 msgid "CREATOR"
-msgstr "Creator:"
-
-msgid "DELETE_PLAYLIST"
-msgstr "Delete Playlist"
-
-msgid "DELETE_PLAYLIST_CONFIRM_1"
-msgstr "Are you sure you want to delete this Playlist?"
-
-msgid "DELETE_PLAYLIST_CONFIRM_2"
-msgstr "This action is irreversible!"
-
-msgid "DELETE_PLAYLIST_CONFIRM_NO"
-msgstr "No"
-
-msgid "DELETE_PLAYLIST_CONFIRM_YES"
-msgstr "Yes"
+msgstr "Skapare:"
 
 msgid "ERROR_AUTHOR_IS_BLOCKED"
-msgstr "You cannot perform this action with this Flipnote because you have blocked the author."
+msgstr "Du kan inte göra detta med denna Flipnote då du har blockerat skaparen."
 
 msgid "ERROR_AUTHOR_IS_BLOCKED_DESC"
-msgstr "Please unblock the author and try again."
+msgstr "Snälla avblockera skaparen och försök igen."
 
 msgid "ERROR_AUTHOR_IS_BLOCKING"
-msgstr "You cannot perform this action with this Flipnote because the author has blocked you."
+msgstr "Du kan inte göra detta med denna Flipnote då skaparen har blockerat dig."
 
 msgid "ERROR_AUTHOR_IS_BLOCKING_DESC"
-msgstr "Please try again with a different Flipnote."
```

Comment:
String for `ERROR_AUTHOR_IS_BLOCKING_DESC` missing, ID for `ERROR_HINT_INCREASE_LIMIT_PLUS` missing